src/test/ui/consts/const_constructor/feature-gate-const_constructor.stderr

```diff
@@ -0,0 +1,34 @@
+error[E0015]: calls in constants are limited to constant functions, tuple structs and tuple variants
+  --> $DIR/feature-gate-const_constructor.rs:5:37
+   |
+LL | const EXTERNAL_CONST: Option<i32> = {Some}(1);
+   |                                     ^^^^^^^^^
+
+error[E0015]: calls in constants are limited to constant functions, tuple structs and tuple variants
+  --> $DIR/feature-gate-const_constructor.rs:7:24
+   |
+LL | const LOCAL_CONST: E = {E::V}(1);
+   |                        ^^^^^^^^^
+
+error[E0723]: can only call other `const fn` within a `const fn`, but `move _2` is not stable as `const fn`
+  --> $DIR/feature-gate-const_constructor.rs:11:13
+   |
+LL |     let _ = {Some}(1);
+   |             ^^^^^^^^^
+   |
+   = note: for more information, see issue https://github.com/rust-lang/rust/issues/57563
+   = help: add #![feature(const_fn)] to the crate attributes to enable
+
+error[E0723]: can only call other `const fn` within a `const fn`, but `move _2` is not stable as `const fn`
+  --> $DIR/feature-gate-const_constructor.rs:16:13
+   |
+LL |     let _ = {E::V}(1);
+   |             ^^^^^^^^^
+   |
+   = note: for more information, see issue https://github.com/rust-lang/rust/issues/57563
+   = help: add #![feature(const_fn)] to the crate attributes to enable
```

Comment:
Can you enable `#![feature(const_fn)]` above to see whether `const_constructor` is suggested?

Reply:
It's not.

Reply:
Hmm that's unfortunate... the user has no way of knowing how to enable the feature gate as-is.